relay-server/src/envelope.rs

```diff
@@ -953,7 +953,7 @@ impl Envelope {
         })
     }
 
-    pub fn trace_context(&self) -> Option<&TraceContext> {
+    pub fn trace_context(&self) -> Option<&DynamicSamplingContext> {
```

Comment:
Not strongly opinionated, but I'd also rename the functions to `dyn_context` or similar. I find it less confusing than having them named `trace_context` and not returning a `TraceContext`.

Reply:
done